django/contrib/admin/tests.py

```diff
@@ -133,17 +133,15 @@ def get_css_value(self, selector, attribute):
         return self.selenium.execute_script(
             'return django.jQuery("%s").css("%s")' % (selector, attribute))
 
-    def get_select_option(self, selector, value):
+    def select_option(self, selector, value):
         """
-        Return the <OPTION> with the value `value` inside the <SELECT> widget
-        identified by the CSS selector `selector`.
+        Select the <OPTION> with the value `value` inside the <SELECT> widget
+        identified by the CSS selector `selector.
```

Comment:
You missed a backtick at the end of the line.

Reply:
Good catch, thanks!

Reply:
You're welcome!